concepticondata/conceptlists/Hantgan-2022-351.tsv

```diff
@@ -0,0 +1,352 @@
+Hantgan-2022-351-276	276	SMALL	ê. petit [Adj kátíyá 1b] [KS kacca ].	1246	SMALL
+Hantgan-2022-351-277	277	SMELL	odeur	2124	SMELL
+Hantgan-2022-351-278	278	SMOKE	fuméé	778	SMOKE (EXHAUST)
+Hantgan-2022-351-279	279	SMOOTH	aplanir, aplatir (surface d’un mur, ou sol d’une maison, avec une truelle.) [<Ful vmeel-].	1234	SMOOTH
+Hantgan-2022-351-280	280	SNAKE	serpent	730	SNAKE
+Hantgan-2022-351-281	281	SOAP	savon	788	SOAP
+Hantgan-2022-351-282	282	SONG	chan	1096	SONG
+Hantgan-2022-351-283	283	SPEAR	caoutchouc, lance-pierres	945	SPEAR
+Hantgan-2022-351-284	284	SPIN	filer (coton)	121	SPIN
+Hantgan-2022-351-285	285	SPIRIT	âme, esprit vital (d'un vivant)	53	SPIRIT
+Hantgan-2022-351-286	286	SPIT	cracher	1440	SPIT
+Hantgan-2022-351-287	287	SPLEEN	rate	1744	SPLEEN
+Hantgan-2022-351-288	288	SPRING OR WELL	(piège) sauter.	2144	SPRING OR WELL
+Hantgan-2022-351-289	289	STAR	etoile	1430	STAR
+Hantgan-2022-351-290	290	STEAL	voler	713	STEAL
+Hantgan-2022-351-291	291	STOMACH	estomac	819	STOMACH
+Hantgan-2022-351-292	292	STONE	petite pierre, caillou, gravier	857	STONE
+Hantgan-2022-351-293	293	STRIKE OR BEAT	gifler, frapper, battre	2133	STRIKE OR BEAT
+Hantgan-2022-351-294	294	SUCK	lécher, sucer	1421	SUCK
+Hantgan-2022-351-295	295	SUGAR	sucre	862	SUGAR
+Hantgan-2022-351-296	296	SUN	soleil	1343	SUN
+Hantgan-2022-351-297	297	SWALLOW	avale	897	SWALLOW
+Hantgan-2022-351-298	298	SWEEP	balayer	230	SWEEP
+Hantgan-2022-351-299	299	SWEET	(goût) doux, sucré	717	SWEET
+Hantgan-2022-351-300	300	SWIM	nager	1439	SWIM
+Hantgan-2022-351-301	301	TAIL	queue	1220	TAIL
+Hantgan-2022-351-302	302	TALK	dire, parler	2708	TALK
+Hantgan-2022-351-303	303	TASTE	goûter	1373	TEA
+Hantgan-2022-351-304	304	TEN	thé	1515	TEN
+Hantgan-2022-351-305	305	TESTICLES	oeuf, testicule	797	TESTICLES
+Hantgan-2022-351-306	306	THICK	(bâton etc.) épais	1244	THICK
+Hantgan-2022-351-307	307	THIGH	cuisse	800	THIGH
+Hantgan-2022-351-308	308	THING	chose	738	THING
+Hantgan-2022-351-309	309	THINK	croire; réfléchir [cf. hò.gù, só:sê, lásá:bû, síkkê] -ex: à.bò .mí:ló [.ò.gú wìy-à/ he thinks they are doing the killing	2271	THINK
+Hantgan-2022-351-310	310	THORN	épine	124	THORN
+Hantgan-2022-351-311	311	THOUSAND	mille	1843	THOUSAND
+Hantgan-2022-351-312	312	THREE	trois	492	THREE
+Hantgan-2022-351-313	313	THROAT	cou, gorge, col	1346	THROAT
+Hantgan-2022-351-314	314	THROW	jeter	1456	THROW
+Hantgan-2022-351-315	315	TIE	attacher	1917	TIE
+Hantgan-2022-351-316	316	TOBACCO	tabac [related verb súndú 1a; SYN:táb-ò ; cf. hámpérê ]	974	TOBACCO
+Hantgan-2022-351-317	317	TOE	doigt, orteil	1389	TOE
+Hantgan-2022-351-318	318	TOMORROW	demain	1329	TOMORROW
+Hantgan-2022-351-319	319	TONGS	pinces (pour enlever les échardes); pinces (du forgeron)	272	TONGS
+Hantgan-2022-351-320	320	TONGUE	lâcher	1205	TONGUE
+Hantgan-2022-351-321	321	TOOTH	dent	1380	TOOTH
+Hantgan-2022-351-322	322	TOUCH	toucher	1892	TOUCH
+Hantgan-2022-351-323	323	TRAP (PITFALL)	piège	1075	TRAP (PITFALL)
+Hantgan-2022-351-324	324	TRAVERSE	casser, détruire ; traverser	666	RIVER
```

Comment:
Is this a mismapping?

Reply:
Completely.

Reply:
I'd leave it unmapped, as it is a complex meaning.